index.js

```diff
@@ -16,10 +17,10 @@ ${title}
 ${favicon}
 ${css}
 ${head}
+${script}
 </head>
 <body>
 ${body}
-${script}
```

Comment:
ah yeah so what I was thinking is that if `scriptAsync` is off, then the script tag is at the bottom of the body tag. If it's enabled, it should be in the head.